cachi2/core/package_managers/gomod.py

```diff
@@ -886,6 +901,10 @@ def _module_lines_from_modules_txt(app_dir: Path) -> List[str]:
     has_packages = {}
 
     log.debug("Parsing modules from vendor/modules.txt")
+    unexpected_format_solution = (
+        "Does `go mod vendor` make any changes to modules.txt?\n"
+        "If not, please let the maintainers know that Cachi2 fails to parse valid modules.txt"
```

Comment:
In this case, "maintainers" is always us (upstream repo). Should we make it clear here that we don't mean administrators?

Reply:
I'd prefer maintainers over authors, but don't have strong opinion on that.

Reply:
Yeah thinking back on it, authors kinda implies you're supposed to go bother the one who wrote the code originally, not the poor guy who's in charge of it now :smile: 

I re-phrased the FetchError message, so I believe "maintainers" always refers to us Cachi2 maintainers now.